config.yaml

```diff
@@ -0,0 +1,3 @@
+sink: glog
```

Comment:
It doesn't seem useful to change default config file format and values here since a custom config file can be passed in with `EVENTROUTER_CONFIG` when deployed.

Reply:
This variable only sets a custom path to the config file.  viper is still dependent on the type to parse it correctly.  However, I will still remove this file since it is not necessary here.